build-package.sh

```diff
@@ -708,6 +714,7 @@ for ((i=0; i<${#PACKAGE_LIST[@]}; i++)); do
 		termux_add_package_to_built_packages_list "$TERMUX_PKG_NAME"
 		termux_step_finish_build
 	) 5< "$TERMUX_BUILD_LOCK_FILE"
+		termux_step_post_finish_build
```

Comment:
In this case `termux_step_post_finish_build` will be executed in parent shell. Move it to `termux_step_finish_build` invocation please.

Also, why exactly `termux_step_post_finish_build` is needed? You can perform checks in `termux_step_post_massage` step.